### ISSUE TITLE
Typos & Request

### ISSUE THREAD

**Issue by zhongxiang117** [schrojunzhang/KarmaDock#17]
* [Doc about total degree](https://github.com/schrojunzhang/KarmaDock/blob/main/dataset/ligand_feature.py#L113), correct `... dim=5` to `... dim=8` or simply remove it.
* [pos_ture](https://github.com/schrojunzhang/KarmaDock/blob/main/architecture/KarmaDock_architecture.py#L56), please search on this phase, it should be `pos_true`, right?

Additionally, can the training scripts be released? Thanks!

**Comment by CLG68** [comment]
other corrections:

In the read.me file:

cd /root/KarmaDock/utils 
python -u virtual_screening_pipeline.py --ligand_smi /root/KarmaDock/DEKOIS2/pdk1/active_decoys.smi --protein_file /root/KarmaDock/DEKOIS2/pdk1/protein/pdk1_protein.pdb --crystal_ligand_file /root/KarmaDock/DEKOIS2/pdk1/protein/pdk1_ligand.mol2 --graph_dir /root/KarmaDock/DEKOIS2/pdk1/karmadock_liggraph --random_seed 2023 

should have this argument: --mode generate_graph

cd /root/KarmaDock/utils 
python -u virtual_screening_pipeline.py --protein_file /root/KarmaDock/DEKOIS2/pdk1/protein/pdk1_protein.pdb --crystal_ligand_file /root/KarmaDock/DEKOIS2/pdk1/protein/pdk1_ligand.mol2 --graph_dir /root/KarmaDock/DEKOIS2/pdk1/karmadock_liggraph --out_dir /root/KarmaDock/DEKOIS2/pdk1/karmadocked --score_threshold 50 --batch_size 64 --random_seed 2023 --out_uncoorected --out_corrected

should have this argument: --mode vs
